ui/src/app/base/components/FormikFormContent/FormikFormContent.tsx

```diff
@@ -161,7 +162,7 @@ const FormikFormContent = <V, E = null>({
       {editable && (
         <FormikFormButtons
           {...buttonsProps}
-          cancelDisabled={saving}
+          cancelDisabled={cancelDisabled === false ? false : saving}
```

Comment:
I can't remember why we made `cancelDisabled` tied to `saving`? I've made it so that you can override that behaviour if you explicitly set `cancelDisabled` to false, but I'm wondering if we should remove the `saving` bit...

Reply:
Also, I meant to say this could probably do with a test.

Reply:
Good call!